### PR TITLE
lawnmowers from hippie

### PR DESCRIPTION
**Pull request by praisenarsie**
credits to erik aka chnkr on hippie for coding this in the firstplace.

:cl: Code by Erik from Hippie, Ported by Frozenguy5
add: Lawnmowers
/:cl:

@vuonojenmustaturska #35144 is an upstream bug i'm telling you, i'm p sure this is for all vehicles that a xeno queen and other mobs can ride all vehicles

### CODE REVIEW

**Comment by praisenarsie** [comment]
@MMMiracles aeiou i'll do it on friday if i remember

**Comment by optimumtact** [comment]
pictures, also I'm not sure why we would need this, we don't have any large grass areas

**Comment by MoreRobustThanYou** [comment]
@optimumtact memes

**Comment by optimumtact** [comment]
that's not a justification, that's an excuse

**Comment by MoreRobustThanYou** [comment]
also, on hippie, traitor botanists can buy a lawnmower that gibs people. most of they time they manage to gib a few before they get murdered.

**Comment by MMMiracles** [comment]
just make it chew through floral/kudzu and slap it in cargo as a more expensive/efficient way to get rid of weeds.

**Comment by PKPenguin321** [comment]
Craft one by taping a chainsaw to a scooter and reinforcing it with plasteel

**Comment by deathride58** [comment]
Speedmerge or riot

**Comment by Bawhoppen** [comment]
One PR at a time

File: code/modules/vehicles/lawnmower.dm [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+			if(M == H)
+				continue
+			if(M.lying)
+				visible_message("<span class='danger'>\the [src] grinds [M.name] into a fine paste!</span>")
+				M.gib()
+				shake_camera(M, 20, 1)
+				gibbed = TRUE
+
+	if(gibbed)
+		shake_camera(H, 10, 1)
```
Comment (deathride58) [inline code/modules/vehicles/lawnmower.dm]: Less is more.

You can probably chop the duration in half here, and keep the strength the same

File: code/modules/vehicles/lawnmower.dm [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+		H = buckled_mobs[1]
+
+	if(emagged)
+		for(var/mob/living/carbon/human/M in loc)
+			if(M == H)
+				continue
+			if(M.lying)
+				visible_message("<span class='danger'>\the [src] grinds [M.name] into a fine paste!</span>")
+				M.gib()
+				shake_camera(M, 20, 1)
```
Comment (deathride58) [inline code/modules/vehicles/lawnmower.dm]: You can probably chop the duration in half here, too, and double the strength

Comment (Tlaltecuhtli) [reply code/modules/vehicles/lawnmower.dm]: will the pilot die if he unbuckles?

Comment (MMMiracles) [reply code/modules/vehicles/lawnmower.dm]: It would be extremely painful.

**Pull request closed without merging**